cmake/protobuf-config.cmake.in

```diff
@@ -1,11 +1,17 @@
 # Version info variables
 set(PROTOBUF_VERSION        "@protobuf_VERSION@")
 set(PROTOBUF_VERSION_STRING "@protobuf_VERSION_STRING@")
+set(PROTOBUF_USE_ZLIB       "@protobuf_WITH_ZLIB@")
+
 
 # Current dir
 get_filename_component(_PROTOBUF_PACKAGE_PREFIX
   "${CMAKE_CURRENT_LIST_FILE}" PATH)
 
+if (PROTOBUF_USE_ZLIB)
+  find_package(ZLIB)
```

Comment:
> I can remove it

No need for remove, I just think that it should be `find_package(ZLIB REQUIRED)` instead of `find_package(ZLIB)`. Without `REQUIRED` there will be no error reported if package will not be found. Also hunterized version of `ZLIB` install `ZLIBConfig.cmake` so final command should be `find_package(ZLIB CONFIG REQUIRED)`. See https://github.com/ruslo/hunter/wiki/pkg.zlib